### PR TITLE
feat: add activation checklist

### PR DESCRIPTION
**Pull request by kappa90**
## Problem
- Implements https://github.com/PostHog/product-internal/pull/338
- There are important actions users could take after the ingestion phase
  - Integrate PostHog on their marketing website
  - Learn which custom events you need to track
  - Learn how to different teams can benefit from PostHog
  - Learn more about apps in PostHog
- Requires https://github.com/PostHog/posthog.com/pull/4137 to be merged

## Changes
- Introduces a series of prompts to expose these additional steps to users
- Adds a "How to be successful with PostHog" button in the Help section, to be able to open these prompts at any time
- Improved the in app promps architecture that has been built before for the product tour.
  - However the product tour has been commented out as it needs additional tests to be deployed robustly, will follow in a future PR

image
image
image
image
image

## How did you test this code?
- Updated tests

### CODE REVIEW

**Comment by kappa90** [comment]
@corywatilo would be nice to get a design approval

@joethreepwood please check if the prompts make sense for you

File: frontend/src/lib/logic/inAppPrompt/inAppPromptLogic.tsx
Comment (pauldambra) [inline frontend/src/lib/logic/inAppPrompt/inAppPromptLogic.tsx:258]: delete rather than comment?

File: frontend/src/mocks/handlers.ts
Comment (pauldambra) [inline frontend/src/mocks/handlers.ts:53]: Are we safe to remove this? mock service worker treats an API route with no mock as failing (I think)

File: posthog/models/prompt.py
Comment (pauldambra) [inline posthog/models/prompt.py:126]: I wonder if there's a better way than commenting out the code here...

Either a tag/branch to keep it easily discoverable. Or a comment so the future traveller understands why there's a block of commented code

**Review by pauldambra — COMMENTED** [review]
Few nits/questions just from a pass-through in GitHub UI

File: posthog/models/prompt.py
Comment (kappa90) [reply posthog/models/prompt.py:126]: You're right, I'll remove it and add it on a different branch!

File: frontend/src/mocks/handlers.ts
Comment (kappa90) [reply frontend/src/mocks/handlers.ts:53]: I implemented this mock endpoint for the in app prompts, as it was using feature flags to run, so it shouldn't be needed I believe.

**Comment by corywatilo** [comment]
No approval needed! 🙃 

Only thing we might want to consider is moving the breadcrumb above the description, so:

< navigation >
Title
Description
[next] button

or

Title
< navigation >
Description
[next] button

Reason being: since these appear in the same location, it'd be nice to not have to move my mouse if the description text height changes. If the buttons are above, they'll be in the same spot every time, which is a nice UX win.

Then having a clear CTA at the bottom (to advance) would be 👌. (Button location should match where we put it in modals.)

**Review by pauldambra — APPROVED** [review]
runs great locally

**Comment by kappa90** [comment]
Visual improvements:

image
image

**Review by joethreepwood — APPROVED** [review]
Based on my extremely long tenure as an engineer, I'm happy to say this code looks acceptable.

The words look good too.

File: frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.scss
Comment (Twixes) [inline frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.scss:42]: Nit: hyphens are usually used in places of spaces in BEM, they're a bit more readable than underscored

File: frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.scss
Comment (Twixes) [inline frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.scss:35]: What do you think about making these `width: 100%` always? Would make the right padding of a button more predictable for users

File: frontend/src/lib/logic/inAppPrompt/inAppPromptLogic.tsx
Comment (Twixes) [inline frontend/src/lib/logic/inAppPrompt/inAppPromptLogic.tsx:102]: Does this constitute a disruption for users already exposed to tooltips?

File: frontend/src/lib/logic/inAppPrompt/inAppPromptLogic.tsx
Comment (Twixes) [inline frontend/src/lib/logic/inAppPrompt/inAppPromptLogic.tsx:91]: Idiomatically:
```suggestion
export enum DefaultAction {
```

File: frontend/src/lib/components/HelpButton/HelpButton.tsx
Comment (Twixes) [inline frontend/src/lib/components/HelpButton/HelpButton.tsx:145]: Hmm, this icon is super old, I think `IconTrendingUp` is our more modern version

File: frontend/src/lib/components/HelpButton/HelpButton.tsx
Comment (Twixes) [inline frontend/src/lib/components/HelpButton/HelpButton.tsx:153]: On the project homepage, this button does the same thing as "Explain this page" 🤔 
Screen Shot 2022-09-26 at 18 10 21

**Review by Twixes — CHANGES_REQUESTED** [review]
I like the change! Though I found a couple of rough edges. I also totally agree with Cory's suggestion that the primary actions, backwards/forwards, should be more prominent

**Comment by kappa90** [comment]
Since https://github.com/PostHog/posthog.com/pull/4137 is finally merged, I fixed the last things according to @Twixes 's review. Good to go if you give me green light.

**Comment by kappa90** [comment]
Any idea why the CH test keeps on failing? Seems completely unrelated to the PR

**Review by Twixes — APPROVED** [review]
LGTM

**Pull request merged**
